packages/core/src/renderer/components/table/table-state.injectable.tsx

```diff
@@ -33,16 +53,15 @@ export function createLensTableState<K extends KubeObject>({
 }: TableDataContextValue<K>, createState: CreateTableState) {
   const headers = renderTableHeader as Required<TableCellProps>[];
 
-
-  let headingColumns: TableDataColumn<K>[] = headers.map(
+  let headingColumns: TableDataColumn[] = headers.map(
     ({ id: columnId, className, title }, index) => ({
       id: columnId ?? className,
       className,
       resizable: !!columnId,
       sortable: !!columnId,
       draggable: !!columnId, // e.g. warning-icon column in pods
       title,
-      renderValue(row: any, col: any) {
+      renderValue(row: TableDataRow) {
```

Comment:
Should be `TableDataRow<K>` otherwise you'll loose `row.data` typing (if provided in `createLensTableState()`